Multi-arch github-action workflow unification (#3200)
containers/buildah: contrib/buildahimage/README.md

The container images are:

  * `quay.io/containers/buildah:<version>` and `quay.io/buildah/stable:<version>` -

>     These images are built when a new buildah version becomes available in

Reviewer: ```suggestion
    These images are built when a new Buildah version becomes available in
```

    Fedora.  These images are intended to be unchanging and stable, they will
    never be updated by automation once they've been pushed.  For build

>     details, see the configuration used to build it,

Reviewer: ```suggestion
    details, see the [buildahimage/stable/Dockerfile](stable/Dockerfile) used to build it.
```

>     [buildahimage/stable/Dockerfile](stable/Dockerfile).

Reviewer: ```suggestion

```

  * `quay.io/containers/buildah:latest` and `quay.io/buildah/stable:latest` -

>     Built daily using the same Containerfile as above.  The buildah version

Reviewer: ```suggestion
    Built daily using the same Dockerfile as above.  The Buildah version
```

    will remain the "latest" available in Fedora, however the other image
    contents may vary compared to the version-tagged images.
  * `quay.io/buildah/testing:latest` - This image is built daily, using the

>     latest version of buildah that was in the Fedora `updates-testing` repository.

Reviewer: ```suggestion
    latest version of Buildah that was in the Fedora `updates-testing` repository.
```
